install/resources/mybb_theme.xml

```diff
@@ -11508,7 +11508,7 @@ if(use_xmlhttprequest == "1")
 		<tr>
 			<td class="thead">
 				<div class="float_right">
-					<span class="smalltext"><strong><a href="javascript:void(0)" id="thread_modes">{$lang->thread_modes}</a>{$threadnoteslink}</strong></span>
+					<span class="smalltext"><strong><a href="showthread.php?mode={$thread_toggle}&amp;tid={$tid}&amp;pid={$pid}#pid{$pid}">{$lang->thread_toggle}</a>{$threadnoteslink}</strong></span>
```

Comment:
This should use the `get_thread_link()` function to build URLs, remember SEO is a core feature.

Reply:
`get_thread_link()` doesn't support custom fragments, like here we use `mode`

Reply:
Yes, we usually build the url in code, it has been suggested to fix this by introducing a global function to handle link building.

But to be frank, I'm unsure if that would be accepted for 1.8, and if not then this PR might as well just be left as is.

@euantorano @dvz